app/src/main/java/de/danoeh/antennapod/fragment/PlaybackHistoryFragment.java

```diff
@@ -258,7 +127,7 @@ protected void afterBindViewHolder(EpisodeItemViewHolder holder, int pos) {
             // played items shouldn't be transparent for this fragment since, *all* items
             // in this fragment will, by definition, be played. So it serves no purpose and can make
             // it harder to read.
-            holder.itemView.setAlpha(1.0f);
+            holder.itemView.findViewById(R.id.container).setAlpha(1.0f);
```

Comment:
BTW I am not sure but I think this actually never worked and I'm not entirely sure we should "fix" it like this.

I've been using the normal app alongside my own debug build that incorporates these changes for a few days now and I realized that the normal app still reflects the played/unplayed nature of the items in the Playback History fragment.

I sometimes find this useful because some elements get in the playback history because I played them by accident, or I listened to half of the episode and I plan to go back later.

Not a big deal either way — another way to fix this might be to use *less* transparency than normal for played items, like `0.8f` if played and 1.0f otherwise.

Reply:
Oh, right. In that case I would just remove the adapter extension here and use the default opacity. Setting up the context menu items can also be done in an anonymous class.

Reply:
> Setting up the context menu items can also be done in an anonymous class.

Can you clarify what you mean by this, or provide an example? This PR is my first bit of Android / Java programming in a *long* time, and I'm not familiar with this particular idiom. Googling some relevant terms isn't turning up anything obvious.

Reply:
I was thinking about something like `listAdapter = new EpisodeItemListAdapter() { void onCreateContextMenu(){} }` but keeping the inner class is fine as well :)